specs/market-api.yaml

```diff
@@ -429,92 +409,114 @@ paths:
+      summary: CounterProposalDemand - Responds with a bespoke Demand to received Offer.
       description: >
         Creates and sends a modified version of original Demand (a
         counter-proposal) adjusted to previously received Proposal (ie. Offer).
         Changes Proposal state to `Draft`. Returns created Proposal id.
       requestBody:
         $ref: '#/components/requestBodies/Proposal'
       responses:
-        '201':
+        201:
           $ref: '#/components/responses/ProposalId'
-          # links: TODO
-        '400':
-          $ref: '#/components/responses/BadRequest'
-        '401':
-          $ref: '#/components/responses/Unauthorized'
-        '404':
-          $ref: '#/components/responses/NotFound'
-        '410':
+        400:
+          $ref: 'common.yaml#/responses/BadRequest'
+        401:
+          $ref: 'common.yaml#/responses/Unauthorized'
+        404:
+          $ref: 'common.yaml#/responses/NotFound'
+        410:
           description: Proposal rejected.
         default:
-          $ref: '#/components/responses/UnexpectedError'
+          $ref: 'common.yaml#/responses/UnexpectedError'
 
+  /demands/{subscriptionId}/proposals/{proposalId}/reject:
+    parameters:
+      - $ref: '#/components/parameters/subscriptionId'
+      - $ref: '#/components/parameters/proposalId'
+    post:
+      tags:
+        - requestor
+      operationId: rejectProposalOfferReason
+      summary: RejectProposalOfferReason - Rejects Proposal (Offer).
+      description: >
+        Effectively ends a Negotiation chain - it explicitly indicates that the sender
+        will not create another counter-Proposal.
+      requestBody:
+        $ref: '#/components/requestBodies/Reason'
+      responses:
+        204:
+          description: Proposal rejected.
+        401:
+          $ref: 'common.yaml#/responses/Unauthorized'
+        404:
+          $ref: 'common.yaml#/responses/NotFound'
+        410:
+          description: Proposal already rejected.
+        default:
+          $ref: 'common.yaml#/responses/UnexpectedError'
 
   /agreements:
     post:
       tags:
         - requestor
       operationId: createAgreement
-      summary: Creates Agreement from selected Proposal.
+      summary: CreateAgreement - Creates Agreement from selected Proposal.
       description: >
         Initiates the Agreement handshake phase.
 
-
         Formulates an Agreement artifact from the Proposal indicated by the
         received Proposal Id. Created Agreement is in `Proposal` state.
 
-
         The Approval Expiry Date is added to Agreement artifact and implies
         the effective timeout on the whole Agreement Confirmation sequence.
 
-
         A successful call to `createAgreement` shall immediately be followed
         by a `confirmAgreement` and `waitForApproval` call in order to listen
         for responses from the Provider.
 
-
         **Note**: Moves given Proposal to `Approved` state.
```

Comment:
I assume it was due to formatting in swagger

Reply:
yes, restoring